tests/Message/SearchRequestTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Omnipay\Braintree\Message;
+
+use Omnipay\Common\Exception\InvalidRequestException;
+use Omnipay\Tests\TestCase;
+
+class SearchRequestTest extends TestCase
+{
+    /**
+     * @var SearchRequest
+     */
+    private $request;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        $this->request = new SearchRequest($this->getHttpClient(), $this->getHttpRequest(), \Braintree_Configuration::gateway());
+        $this->request->initialize(
```

Comment:
This is a bit picky, but I feel this level of fixture setup complicates subsequent tests. That it's fine to use `setUp()` for bootstrapping the basic instance of request... but that in this situation, the `initialize()` calls should probably belong in the tests themselves. 

For me, using `initialize()` in `setUp()` creates a couple of issues:

1. it makes tests like `testGetData()` a bit disconnected… where did the `2187588535 ` come from.
2. it complicates the `testValidateWillFailWIthNoOrderNumber()` test by forcing it to repeat the setup process again.